tests/channels/pjsip/dialplan_functions/pjsip_notify/configs/ast1/extensions.conf

```diff
@@ -0,0 +1,28 @@
+[general]
+static=yes
+writeprotect=yes
+autofallthrough=yes
+clearglobalvars=no
+priorityjumping=yes
+
+[globals]
+
+[default]
+exten => 1212,1,noOp()
+    same => n,Answer()
+    same => n,PJSIPNotify(,&Event=Test&X-Data=Fun)
+    same => n,Wait(3)
+    same => n,Hangup()
+
+exten => 2323,1,noOp()
+    same => n,Answer()
+    same => n,PJSIPNotify(,test-notify)
+    same => n,Wait(3)
+    same => n,Hangup()
+
+exten => 3434,1,noOp()
+    same => n,Answer()
+    same => n,Wait(1)
+    same => n,PJSIPNotify(<sip:bob@127.0.0.1:5260>,&Event=Test&X-Data=Fun)
+    same => n,Wait(3)
+    same => n,Hangup()
```

Comment:
I think you need a test for setting the Notify body from the dialplan app.

Reply:
added